### ISSUE TITLE
Unexpected token < in JSON at position 0 -> mvg.getRoute()

### ISSUE THREAD

**Issue by lostdesign** [lukaskollmer/mvg-node#2]
Hi,

i am using `mvg.getRoute()` sometimes it's working fine and sometimes it's throwing `Uncaught SyntaxError: Unexpected token < in JSON at position 0`. The stations i tried were `Hauptbahnhof` -> `Ostbahnhof.`

```js
// vue component
async getRoutesOut(){
  let routesOut = 
    await mvg.getRoute(this.route.from, this.route.to)
    .then(res =>{ 
        this.routesOut = res;
    })
    .catch(err => console.log(err));
},
```

```
VM553:1 Uncaught SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at Request._callback (C:\Users\user\Documents\code\mvg\node_modules\mvg-node\index.js:16)
    at Request.self.callback (C:\Users\user\Documents\code\mvg\node_modules\request\request.js:185)
    at Request.emit (events.js:194)
    at Request.<anonymous> (C:\Users\user\Documents\code\mvg\node_modules\request\request.js:1161)
    at Request.emit (events.js:194)
    at IncomingMessage.<anonymous> (C:\Users\user\Documents\code\mvg\node_modules\request\request.js:1083)
    at Object.onceWrapper (events.js:282)
    at IncomingMessage.emit (events.js:199)
    at endReadableNT (_stream_readable.js:1141)
```

https://github.com/lukaskollmer/mvg-node/blob/de9207666827a3fb75a0ae268aa34576ca1a38dd/index.js#L16

It seems like it's due to MVG having issues, so maybe there might be a need to catch their error message:

```html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="Content-Language" content="de">
  <title>apps.mvg-fahrinfo.de - Error 500, internal error - ooops</title>
  <meta http-equiv="Content-Type"
 content="text/html; charset=iso-8859-1">
  <meta http-equiv="Content-Language" content="de">
</head>
<body style="color: rgb(0, 0, 0); background-color: rgb(255, 255, 255);">
<font> </font>
<p>&nbsp;</p>
<p>&nbsp;</p>
<table style="border-collapse: collapse;" id="AutoNumber1" border="0"
 bordercolor="#111111" cellpadding="0" cellspacing="0" width="100%">
  <tbody>
    <tr>
      <td width="30%"> <br>
      </td>
      <td width="70%"><font face="Arial">
      <h2> 500 - interner Fehler</h2>
      <p>Leider ist bei der Verarbeitung Ihrer Anfrage ein interner
Fehler aufgetreten.<br>
Bitte versuchen Sie den Zugriff in einigen Minuten erneut.<br>
      </p>
      <p>Wir entschuldigen uns f&uuml;r evtl. auftretende
Unannehmlichkeiten und<br>
bitten um Ihr Verst&auml;ndnis.<br>
      </p>
      <p allign="center">&nbsp;</p>
      </font>
      <p>&nbsp; </p>
      </td>
    </tr>
  </tbody>
</table>
</body>
</html>
````

**Comment by lukaskollmer** [comment]
Gotta be honest i don't really use the routing api a lot, that's probably why i never added any sort of error handling (or even encountered that error in the first place).

I published a new version (v1.2.3) which returns null if the api didn't return valid JSON, that should fix this for the time being.

**Comment by lostdesign** [comment]
Thanks anyhow - if i will find some time, i will look into this and open a PR.
